opt_PXL.ino

```diff
@@ -1,17 +1,18 @@
-// Adafruit NeoPixel - Version: 1.1.6
-#include <Adafruit_NeoPixel.h>
+// SD - Version: 1.2.2
+#include <SD.h>
 
-// Adafruit_SSD1306-1.1.2 - Version: Latest 
+// Adafruit SSD1306 - Version: 1.1.2
 #include <Adafruit_SSD1306.h>
 
-// SD - Version: Latest 
-#include <SD.h>
+// Adafruit NeoPixel - Version: 1.1.6
+#include <Adafruit_NeoPixel.h>
+
 
 
   
 // Pin assignments for the Arduino (Make changes to these if you use different Pins)
-#define SDssPin 53                        // SD card CS pin
-byte NPPin = 31;                           // Data Pin for the NeoPixel LED Strip
+#define SDssPin 7                        // SD card CS pin
+byte NPPin = 8;                           // Data Pin for the NeoPixel LED Strip

```

Comment:
@woodsy900 I assume you meant to make these pin changes?

Reply:
Yes, the pin changes as it suggests in the comments tell the program where the SD Clock Source pin is (pin 7 on the nano) and where the Green Data wire is attached to the nano (Pin 8 or D8)